lib/Sema/TypeCheckConstraints.cpp

```diff
@@ -1466,10 +1465,10 @@ void ConstraintSystem::print(raw_ostream &out) const {
              });
   for (auto tv : typeVariables) {
     out.indent(2);
-    tv->getImpl().print(out);
```

Comment:
Removing this from will end up with " equivalent to $T0" in case type variable is not a representative. Is that correct?

Reply:
This has been fixed now.